esri/build_opencv_windows.sh

```diff
@@ -0,0 +1,83 @@
+function cmake_configure
+{
+  cmake \
+    -G "Visual Studio 17 2022" \
+    -T "version=14.33.31629" \
+    -DCMAKE_INSTALL_PREFIX=do_not_install_here \
+    -DOPENCV_PYTHON_SKIP_DETECTION=ON \
+    -DWITH_FFMPEG=OFF \
+    -DWITH_OPENCL=OFF \
+    -DWITH_JPEG=OFF \
+    -DWITH_JASPER=OFF \
+    -DWITH_IPP=OFF \
+    -DWITH_OPENEXR=OFF \
+    -DWITH_OPENJPEG=OFF \
+    -DWITH_PNG=OFF \
+    -DWITH_PROTOBUF=OFF \
+    -DWITH_TIFF=OFF \
+    -DWITH_WEBP=OFF \
+    -DBUILD_opencv_apps=OFF \
+    -DBUILD_opencv_features2d=OFF \
+    -DBUILD_opencv_flann=OFF \
+    -DBUILD_opencv_core=ON \
+    -DBUILD_opencv_imgcodecs=OFF \
+    -DBUILD_opencv_imgproc=ON \
+    -DBUILD_opencv_apps=OFF \
+    -DBUILD_opencv_calib3d=OFF \
+    -DBUILD_opencv_dnn=OFF \
+    -DBUILD_opencv_features2d=OFF \
+    -DBUILD_opencv_flann=OFF \
+    -DBUILD_opencv_gapi=OFF \
+    -DBUILD_opencv_highgui=OFF \
+    -DBUILD_opencv_java_bindings_generator=OFF \
+    -DBUILD_opencv_js=OFF \
+    -DBUILD_opencv_js_bindings_generator=OFF \
+    -DBUILD_opencv_ml=OFF \
+    -DBUILD_opencv_objc_bindings_generator=OFF \
+    -DBUILD_opencv_objdetect=OFF \
+    -DBUILD_opencv_photo=OFF \
+    -DBUILD_opencv_python_bindings_generator=OFF \
+    -DBUILD_opencv_python_tests=OFF \
+    -DBUILD_opencv_stitching=OFF \
+    -DBUILD_opencv_ts=OFF \
+    -DBUILD_opencv_video=OFF \
+    -DBUILD_opencv_videoio=OFF \
+    -DBUILD_opencv_world=OFF \
+    "$@"
+}
+
+set -ex
+
+rm -rf build_x64
+rm -rf install_x64
+cmake_configure -S opencv-4.7.0 -B build_x64 -A x64
```

Comment:
do `CMAKE_SYSTEM_NAME`  `CMAKE_SYSTEM_VERSION` `CMAKE_SYSTEM_PROCESSOR` need to be set?

Reply:
Not if you are building on a windows_x64 machine. I'm not sure if its possible to build for x64 on a x86.